pkg/tsdb/cloudwatch/log_actions_test.go

```diff
@@ -0,0 +1,178 @@
+package cloudwatch
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/tsdb"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestLogActions(t *testing.T) {
+	Convey("TestLogActions", t, func() {
```

Comment:
We're supposed to migrate from goconvey to the standard testing library + Testify. As such, we shouldn't write new tests with goconvey.

Reply:
Resolving this, since after discussing with @bergquist we decided to focus on the hard issues.